From 18cdaaa482121aef942585cfbb02b1a9058e553d Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Mon, 23 Oct 2023 09:10:54 +0100
Subject: [PATCH] iio: imu: Fix spelling mistake "accelrometer" ->
 "accelerometer"

There are two spelling mistakes in dev_err messages. Fix them.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Reviewed-by: Jagath Jog J <jagathjog1996@gmail.com>
Link: https://lore.kernel.org/r/20231023081054.617292-1-colin.i.king@gmail.com
Signed-off-by: Jonathan Cameron <Jonathan.Cameron@huawei.com>
---
 drivers/iio/imu/bmi323/bmi323_core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/iio/imu/bmi323/bmi323_core.c b/drivers/iio/imu/bmi323/bmi323_core.c
index 0bd5dedd9a63..183af482828f 100644
--- a/drivers/iio/imu/bmi323/bmi323_core.c
+++ b/drivers/iio/imu/bmi323/bmi323_core.c
@@ -545,7 +545,7 @@ static int bmi323_tap_event_en(struct bmi323_data *data,
 	guard(mutex)(&data->mutex);
 
 	if (data->odrhz[BMI323_ACCEL] < 200) {
-		dev_err(data->dev, "Invalid accelrometer parameter\n");
+		dev_err(data->dev, "Invalid accelerometer parameter\n");
 		return -EINVAL;
 	}
 
@@ -1453,7 +1453,7 @@ static int bmi323_enable_steps(struct bmi323_data *data, int val)
 
 	guard(mutex)(&data->mutex);
 	if (data->odrhz[BMI323_ACCEL] < 200) {
-		dev_err(data->dev, "Invalid accelrometer parameter\n");
+		dev_err(data->dev, "Invalid accelerometer parameter\n");
 		return -EINVAL;
 	}
 
